include/mega/logging.h

```diff
@@ -150,13 +182,18 @@ class SimpleLogger
     static OutputMap outputs;
     static OutputStreams getOutput(enum LogLevel ll);
 #else
-    std::array<char, 256> mBuffer; // will be stack-allocated since SimpleLogger is stack-allocated
-    std::array<char, 256>::iterator mBufferIt;
-    using DiffType = std::array<char, 256>::difference_type;
+    const static int LOGGER_CHUNKS_SIZE = 1024;
```

Comment:
On Android your stack is only 8KB

Reply:
we should bear in mind if using PERFORMANCE_MODE for android